python/glow/gwas/lin_reg.py

```diff
@@ -174,7 +167,8 @@ def _create_one_YState(Y: NDArray[(Any, Any), Float], phenotype_df: pd.DataFrame
 def _linear_regression_inner(genotype_pdf: pd.DataFrame, Y_state: YState,
                              Y_mask: NDArray[(Any, Any), Float], Y_scale: NDArray[(Any, ), Float],
                              Q: NDArray[(Any, Any), Float], dof: int,
-                             phenotype_names: pd.Series) -> pd.DataFrame:
+                             phenotype_names: pd.Series, Y_raw: NDArray[(Any, Any), Float] = None,
```

Comment:
This is a private function, so we can change the APIs.

Reply:
I've removed the default values, but this function is still called in a unit test.